### PR TITLE
Migrate from legacy Astronomer products

### PR DESCRIPTION
**Pull request by jwitz**
Putting on draft until we get approval on general wording/ framing

Resolves https://github.com/astronomer/docs/issues/2544

### CODE REVIEW

File: astro/migrate-astronomer-software-nebula.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+---
+title: 'Migrate from Astronomer Software or Nebula to Astro'
+sidebar_label: 'From Astronomer Software/ Nebula'
+id: migrate-to-astro
```
Comment (jwitz) [inline astro/migrate-astronomer-software-nebula.md:4]:
```suggestion
id: migrate-astronomer-software-nebula
```

File: astro/migrate-astronomer-software-nebula.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+title: 'Migrate from Astronomer Software or Nebula to Astro'
+sidebar_label: 'From Astronomer Software/ Nebula'
+id: migrate-astronomer-software-nebula
+---
+
+Astro includes most of the same features that are available in Astronomer Software and Astronomer Nebula and Software. After Astronomer completes your migration, use this document to familiarize yourself with Astro and relocate key workflows and features from other Astronomer products. 
+
+## Prerequisites 
+
+- You have finished migrated your Airflow Deployments to Astro, or you're in the process of migrating.
```
Comment (jwitz) [inline astro/migrate-astronomer-software-nebula.md:11]:
```suggestion
- You have finished migrating your Airflow Deployments to Astro, or you're in the process of migrating. 
```

**Review by fritz-astronomer — CHANGES_REQUESTED** [review]
- Would like to mention Starship to assist with the process
- Should maybe not mention nebula (as that all is being automated today)
- May need to start mentioning Hybrid

Is a good doc as-is, though, at least as an introduction and a "what's changed"

**Comment by jwitz** [comment]
Closing this out as there are no more legacy products to migrate from!

**Pull request closed without merging**